app/assets/javascripts/helpers/customization_helpers.js

```diff
@@ -0,0 +1,17 @@
+function shouldDisplay(element, schoolContext) {
```

Comment:
Oooooh I like this approach! If I'm reading it right this is similar to the the "centralize decision-making methods into one place" pattern that @kevinrobinson has talked about for authorization.

Reply:
Yeah, depending on customization needs, we likely need a more robust solution that lives in a data store, not code. But for the moment this gets us a step in a good direction.

Reply:
👍 👞 👠 <--- those are steps :)